charon/src/ast/types.rs

```diff
@@ -370,7 +370,7 @@ pub struct Variant {
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Field {
-    pub span: Span,
+    pub item_meta: ItemMeta,
```

Comment:
Please add a comment like `FIXME: define a more appropriate container for attribute and visibility information.`